lib/api/2.0/files.js

```diff
@@ -4,6 +4,7 @@
 
 var injector = require('../../../index.js').injector;
 var controller = injector.get('Http.Services.Swagger').controller;
+var files = injector.get('Http.Services.Api.Files');
```

Comment:
Should this extend to renaming "files-api-service.js" to something like "staticfiles-api-service.js"?